--- libesmtp-1.0.6/configure.ac.bero	2013-01-15 23:24:26.934031125 +0100
+++ libesmtp-1.0.6/configure.ac	2013-01-15 23:24:41.965858234 +0100
@@ -24,9 +24,9 @@ dnl Foundation, Inc., 59 Temple Place, S
 dnl 
 dnl #########################################################################
 
-AC_INIT(smtp-api.c)
-AM_CONFIG_HEADER(config.h)
-AM_INIT_AUTOMAKE([libesmtp],[1.0.6])
+AC_INIT([libesmtp],[1.0.6])
+AC_CONFIG_HEADERS([config.h])
+AM_INIT_AUTOMAKE
 AC_CANONICAL_HOST
 AC_CONFIG_MACRO_DIR([m4])
 
